### PR TITLE
Nav bar

### PR DESCRIPTION
**Pull request by enzomaltese**

### CODE REVIEW

**Comment by enzomaltese** [comment]
LGTM

File: frontend/src/components/IntegrationsCarouselIcons.tsx
Comment (fowdre) [inline frontend/src/components/IntegrationsCarouselIcons.tsx:1]: This file is not used anymore, we should probably remove it.

File: frontend/src/components/ui/carousel.tsx
Comment (fowdre) [inline frontend/src/components/ui/carousel.tsx:1]: You should remove this file

**Review by fowdre — CHANGES_REQUESTED** [review]

File: frontend/src/components/IntegrationsCarouselIcons.tsx
Comment (enzomaltese) [reply frontend/src/components/IntegrationsCarouselIcons.tsx:1]: Fix last commit

File: frontend/src/components/ui/carousel.tsx
Comment (enzomaltese) [reply frontend/src/components/ui/carousel.tsx:1]: Fix last commit

File: frontend/src/components/IntegrationsCarouselIcons.tsx
Comment (fowdre) [reply frontend/src/components/IntegrationsCarouselIcons.tsx:1]: LGTM 👍

File: frontend/src/components/ui/carousel.tsx
Comment (fowdre) [reply frontend/src/components/ui/carousel.tsx:1]: LGTM 👍

**Review by fowdre — APPROVED** [review]

**Pull request merged**